test/parachain/src/service.rs

```diff
@@ -72,10 +67,12 @@ macro_rules! new_full_start {
 ///
 /// This function blocks until done.
 pub fn run_collator(
-	parachain_config: Configuration,
+	mut parachain_config: Configuration,
 	key: Arc<CollatorPair>,
 	polkadot_config: polkadot_collator::Configuration,
 ) -> sc_service::error::Result<impl AbstractService> {
+	parachain_config.announce_block = false;
```

Comment:
This should be moved into some function provided by Cumulus. As people will need to replicate this all the time and we may need to "tweak" the config even more in the future.

Reply:
Should I create a crate `service` for this?

Reply:
Ok as discussed I placed it in the `collator` crate